### ISSUE TITLE
Link to Grafana is Not Working

### ISSUE THREAD

**Issue by dmitrii-didenko** [kubecost/cost-analyzer-helm-chart#1651]
**Describe the bug**
Links to Grafana is not available when using configuration with existing Grafana installation
<img width="772" alt="Screenshot 2022-08-22 at 11 40 03" src="https://user-images.githubusercontent.com/81171763/185878250-c90c9ba9-d477-4cc6-b099-5963098c7233.png">

Please note, everything was working fine in 1.94.3, the issue has started to be observed after 1.95.

**To Reproduce**
Deploy helm chart with the following config:
```yaml
readonly: true
global:
  prometheus:
    enabled: false
    fqdn: http://monitoring-prometheus-oper-prometheus.monitoring.svc:9090

  grafana:
    enabled: false
    proxy: false

grafana:
  sidecar:
    dashboards:
      enabled: false
    datasources:
      enabled: false

prometheus:
  nodeExporter:
    enabled: false
  kubeStateMetrics:
    enabled: false

persistentVolume:
  size: 1M
  storageClass: efs-operations-dynamic

# Kubecost settings
kubecostProductConfigs:
  clusterName: "[OUR CLUSTER NAME]"
  grafanaURL: "[URL FOR EXISTING GRAFANA]"
```

**Expected behavior**
Link to grafana should be working fine

As far as I understand, since 1.95 the new frontend was introduced. Should the helm config be change somehow to get links working as it was in 1.94.3?

**Comment by keithhand** [comment]
Hello @dmitrii-didenko, v1.96 should have a fix for this issue. Can you try to upgrade and see if you are still experiencing it within your installation?

**Comment by dmitrii-didenko** [comment]
@keithhand I tried, the issue is still observed. The screenshot in the original post is from 1.96.0
```bash
kdp kubecost-cost-analyzer-85fbdd48b7-44c7b | grep Image:
    Image:          gcr.io/kubecost1/cost-model:prod-1.96.0
    Image:          gcr.io/kubecost1/frontend:prod-1.96.0
```

**Comment by kaelanspatel** [comment]
Hi @dmitrii-didenko, thanks for reporting this.

I think I see the issue; it looks like it's related to how it reads in the helm configs.

Just curious, did that value for `global.grafana.enabled` change at all between your upgrade?

**Comment by dmitrii-didenko** [comment]
@kaelanspatel There was no changes in helm values, just upgraded the version from 1.94.3 to 1.96.0 and the links stopped working

**Comment by kaelanspatel** [comment]
Ok thank you @dmitrii-didenko, will get this looked at.

**Comment by Adam-Stack-PM** [comment]
Linking for reference to help troubleshoot https://github.com/kubecost/cost-analyzer-frontend/issues/903

**Comment by dmitrii-didenko** [comment]
@kaelanspatel hey! I see it's tagged 1.97. Just checked the issue still persist in 1.97, could you please clarify when we can expect the fix? Thanks

**Comment by Adam-Stack-PM** [comment]
@AjayTripathy Could you help trouble shoot this issue?

**Comment by Adam-Stack-PM** [comment]
@dmitrii-didenko,This is on my radar and will follow up next week.

**Comment by AjayTripathy** [comment]
Hey all, the eng team is a bit slammed heading into kubecon-- not sure we expect this to make 1.98 but will reevaluate in 1.99

**Comment by dwbrown2** [comment]
@AjayTripathy @nealormsbee have we repro'd at this point?

**Comment by AjayTripathy** [comment]
I haven't looked-- I wasn't able to repro after 1.97. I'm not sure I'd call this release blocking at this point since this is just a deeplink to grafana, but definitely worth trying again

**Comment by Adam-Stack-PM** [comment]
@dwbrown2, In nightly the links are now working. 

However in the new beta UI we no longer display the referenced view. Not sure from a product side if that was intentional or an additional item we need to add to the new UI.  

Would love you input.

**Comment by dwbrown2** [comment]
@Adam-Stack-PM do you know why we removed those? I added some UI commentary to 1.100 doc.

**Comment by nealormsbee** [comment]
### Grafana links

I haven't tried to repro but it does seem like a bug. At one point, we surfaced `grafanaEnabled` to the FE as `true` if *either* `grafanaEnabled` was true in the helm chart, or a custom grafana URL (pointing to an external) was set.
Now we just send the environment variable directly to the FE, but it's still acting like that bool encodes information about both the state of `.enabled` and/or a custom URL. I can submit a fix for this.


### Links not present in the new UI

The request-sizing UI was redesigned not to include this table, since most pods are managed at the controller-level. I know that at least Webb and maybe Ajay had intuitions that this table is important to keep, though I don't remember why (this conversation was right before holiday break). Will send out an email to get a decision about that UI, but also interested in commentary from our users.

@jjarrett21 this is a good issue to have on your radar for flipping the "default UI experience" switch -- do we need the new UI to have this Request Sizing table to make that switch?

EDIT:

I misspoke above. It's not that most pods are managed at the controller level. It's that our *sizing recommendations* are generated at the *pod* level. So recommendations for all the container instances are the same. Usage can vary, and request *can* vary, but is unlikely to vary given it is set at a higher level.

**Comment by nealormsbee** [comment]
Closed by https://github.com/kubecost/cost-analyzer-frontend/pull/1593

**Comment by dmitrii-didenko** [comment]
@nealormsbee just to get it right, which release this will be available in?

**Comment by nealormsbee** [comment]
@dmitrii-didenko this will be fixed in v1.100, which is scheduled for release next week!
